sys/linux/socket_tipc_netlink.txt

```diff
@@ -93,6 +93,14 @@ tipc_name_table_query {
 	upbound		int32be
 }
 
+type alg_aead_name_tipc string["gcm(aes)", TIPC_AEAD_ALG_NAME]
+
+tipc_aead_key {
+	alg_name	alg_aead_name_tipc
+	keylen		len[key, int32]
+	key		array[int8[TIPC_AEAD_KEYLEN_MIN:TIPC_AEAD_KEYLEN_MAX]]
```

Comment:
This means array of ints in the range [TIPC_AEAD_KEYLEN_MIN, TIPC_AEAD_KEYLEN_MAX].

I think you want `array[int8, TIPC_AEAD_KEYLEN_MIN:TIPC_AEAD_KEYLEN_MAX]`.

Reply:
Yeah my bad. Thanks for catching this. I've just modified it correspondingly.

Reply:
@JiahengHu, can you click "Resolve conversation" when you address code review comments?  Otherwise reviewers have to go through the comments to see what is left.  Thanks!